CONTRIBUTING.md

```diff
@@ -0,0 +1,234 @@
+These issues are great for developers who want to reduce their chances of falling down a rabbit hole in search of a solution.
+
+Of course, you can work on any other issue you like—the "good first" and "ready to take" issues are simply those where the scope and timeline may be better defined.
+Pull requests for other issues, or completely novel problems, may take a bit longer to review if they don't fit into our current development cycle.
+
+If you decide to fix an issue, please make sure to check the comment thread in case somebody is already working on a fix.
+If nobody is working on it at the moment, please leave a comment stating that you've started to work on it, so other people don't accidentally duplicate your effort.
+
+If somebody claims an issue but doesn't follow up after more than a week, it's fine to take over, but you should still leave a comment.
+If there has been no activity on the issue for 7 to 14 days, then it's safe to assume that nobody is working on it.
+
+## Sending a pull request
+
+Pigment CSS as a projects is community-driven, so pull requests are always welcome, but before working on a large change, it's best to open an issue first to discuss it with the maintainers.
+
+When in doubt, keep your pull requests small.
+For the best chances of being accepted, don't bundle more than one feature or bug fix per PR.
+It's often best to create two smaller PRs rather than one big one.
+
+1. Fork the repository.
+
+2. Clone the fork to your local machine and add the upstream remote:
+
+```bash
+git clone https://github.com/<your username>/pigment-css.git
+cd pigment-css
+git remote add upstream https://github.com/mui/pigment-css.git
+```
+
+<!-- #default-branch-switch -->
+
+3. Synchronize your local `master` branch with the upstream one:
+
+```bash
+git checkout master
+git pull upstream master
+```
+
+4. Install the dependencies with pnpm (yarn or npm aren't supported):
+
+```bash
+pnpm install
+```
+
+5. Create a new topic branch:
+
+```bash
+git checkout -b my-topic-branch
+```
+
+6. Make changes, commit, and push to your fork:
+
+```bash
+git push -u origin HEAD
+```
+
+7. Go to [the repository](https://github.com/mui/pigment-css) and open a pull request.
+
+The core team actively monitors for new pull requests.
+We will review your PR and either merge it, request changes to it, or close it with an explanation.
+
+### Trying changes in the apps
+
+While the documentation is still under development, we have few apps in the repository where you can test the changes. Before running the apps, make sure that the packages are build by running
+
+```bash
+pnpm build
+```
+
+After this, you can run one of the applications located in the `/apps` directory, by following their `README.md` files.
+
```

Comment:
How about we delete all of this and instead apply https://github.com/mui/base-ui/blob/master/CONTRIBUTING.md?plain=1? It doesn't feel like we have bandwidth to maintain a hybrid version of this file, well.